mcs/class/Makefile

```diff
@@ -77,6 +77,52 @@ monotouch_runtime_dirs := \
 	System.XML \
 	Mono.CSharp
 
+xammac_4_5_dirs := \
+	corlib	\
+	System	\
+	Mono.Posix			\
+	System.Core	\
+	System.XML	\
+	Mono.Security	\
+	System	\
+	I18N		\
+	System.ServiceModel.Internals	\
+	SMDiagnostics	\
+	System.Numerics	\
+	Mono.Data.Tds	\
+	System.Transactions	\
+	System.EnterpriseServices	\
+	System.Data	\
+	System.Runtime.Serialization	\
+	System.Xml.Linq	\
+	Mono.Data.Sqlite	\
+	System.Runtime.Serialization.Formatters.Soap \
+	System.ComponentModel.DataAnnotations \
+	System.Web.ApplicationServices	\
```

Comment:
^ never used but that sound like a server-side assembly that we should not ship. Does anything depend on it ?